src/styles/color.css

```diff
@@ -10,6 +10,7 @@
   --fiord: #505573;
   --mystic: #d2d6e7;
   --brown: #413e3c;
+  --malibu: #d7f6fc;
```

Comment:
Let's keep the human-readable naming - `cyan`

Reply:
Updated the color